.golangci.yml

```diff
@@ -0,0 +1,52 @@
+run:
+  timeout: 10m
+
+linters:
+  enable:
+    - bodyclose
+    - gofmt
+    - goimports
+    - golint
+    - gosec
+    - misspell
+    - unconvert
+    - unparam
+
+linters-settings:
+  misspell:
+    ignore-words:
+      - clas
+      - cancelled
+    locale: US
+  gofmt:
+    simplify: true
+  unparam:
+    check-exported: false
+
+issues:
+  exclude:
+    # TODO(jpeach): exclude unparam warnings about functions that always recieve
+    # the same arguments. We should clean those up some time.
+    - always receives
+  exclude-rules:
+    - path: zz_generated
+      linters:
+        - goimports
+    # TODO(youngnick): Disable the deprecation warnings, because of the "github.com/golang/protobuf/proto"
+    # libraries. We can't stop using them yet, because they move from Proto v1 to V2
+    # And we can't nolint them by line, because https://github.com/golangci/golangci-lint/issues/741
+    - linters:
+        - staticcheck
+      text: "SA1019: package github.com/golang/protobuf"
+    - linters:
+        - golint
+      text: "var `ingress_https` should be `ingressHTTPS`"
+    - linters:
+        - golint
+      text: "var `ingress_http` should be `ingressHTTP`"
```

Comment:
Are these applicable to the operator?

Reply:
nope, I just copy/pasted from Contour, I removed them.